setup.py

```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-submissions',
-    version='0.1.0',
+    version='1.0',
```

Comment:
@nedbat Just like Django Big jump :). btw this is just for now, we have to update versions when we do final rebase.